src/app/sign_up/sign_up.html

```diff
@@ -0,0 +1,36 @@
+<section class="sign_up">
+  <h1 class="sign-up-header">sign up</h1>
+  <div class="form-container">
+    <form name="signUpForm" ng-submit="signUpForm.$valid && signUpCtrl.createUser()" novalidate>
+      <label for="name" class="form-label" ng-hide="(signUpForm.$submitted || signUpForm.name.$touched) && signUpForm.name.$error.required">name</label>
+      <label for="name" class="form-label error" ng-show="(signUpForm.$submitted || signUpForm.name.$touched) && signUpForm.name.$error.required">name is required</label>
+      <input class="sign-up-input" type="text" name="name" ng-model="signUpCtrl.user.name" required/>
```

Comment:
I see that for all the inputs and labels on the form you are applying the classes `form-label` and `sign-up-input` in order to keep the html cleaner, you could add set styles like:

`.sign-up-form {
     label {.....}
     input {.....}
}`